lib/utils.js

```diff
@@ -279,7 +282,13 @@ function prettifyObject ({
 
     if (keyValue === undefined) continue
 
-    const lines = stringifySafe(input[keyName], null, 2)
+    let stringify = () => stringifySafe(keyValue, null, 2)
+    const customPrettifier = customPrettifiers[keyName]
+    if (typeof customPrettifier === 'function') {
+      stringify = () => customPrettifier(keyValue, keyName, input)
+    }
+
+    const lines = stringify()
```

Comment:
There is no need to create a new function to call it here, just call one of the two inside an if-else.

Reply:
:+1